docs/topics/auth/customizing.txt

```diff
@@ -190,11 +195,11 @@ exception in :meth:`~django.contrib.auth.models.User.has_perm()` or
 :meth:`~django.contrib.auth.models.User.has_module_perms()`, the authorization
 will immediately fail and Django won't check the backends that follow.
 
-The simple backend above could implement permissions for the magic admin
-fairly simply::
+A backend could implement permissions for the magic admin fairly simply::
```

Comment:
magic admin?

Reply:
Not sure either. I just removed the "above" part. Do you have a suggestion?

Reply:
now It seems OK to me.